test/lib/muncher_api_wrapper_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'test_helper'
+
+describe MuncherApiWrapper do
+  describe 'list_recipes' do
+
+    it 'gives a list of recipes' do
+      VCR.use_cassette('recipes') do
+        recipes = MuncherApiWrapper.list_recipes("food")
```

Comment:
What if `list_recipes` produces 0 results? What if `show` is given an invalid URI?